Alert/alert.css

```diff
@@ -0,0 +1,81 @@
+body{
+    font-family: 'Quicksand', sans-serif;
+    font-size: 20px;
+}
+
+.alert-container{
+    border-radius: 1rem;
+    box-shadow:inset 0rem 0rem 0.5rem gray;
+    padding: 1rem;
+    margin: 0.5rem;
+}
+
+h2{
+    margin: 0rem 1rem;
+}
+
+.alert-primary{
+    background-color: rgb(222, 184, 135);
```

Comment:
Use variables for colors

Reply:
these colors are used only once so that is why I didn't made variables for these colors